src/ctia/http/routes/indicator.clj

```diff
@@ -35,26 +35,44 @@
     :tags ["Indicator"]
     (GET "/:id/judgements" []
       :return [StoredJudgement]
-      :path-params [id :- Long]
+      :path-params [id :- s/Str]
+      :query [params SightingsByIndicatorQueryParams]
       :summary "Gets all Judgements associated with the Indicator"
-      (not-found))
+      (if-let [indicator (read-indicator @indicator-store id)]
+        (if-let [judgements (list-judgements-by-indicators @judgement-store [indicator] params)]
```

Comment:
While writing these I would have preferred to have a `list-X-by-indicators-ids` instead.
So yes I am with you here.

Do you have something like this in mind?:

~~~
(let [indicator (read-indicator @indicator-store id)
      judgement-ctia-ids (map #(-> % :judgement_id make-ctia-id) (:judgements indicator))]
    (if-let [jugements (list-judgements-by-indicator-ids @judgement-stor judgement-ctia-ids params)]
...
~~~

where for now I don't really know how to write `make-ctia-id`